MHVLib/HVMobile/HealthVaultResponse.m

```diff
@@ -18,18 +18,12 @@
 
 #import "MHVCommon.h"
 #import "HealthVaultResponse.h"
-#import "XmlTextReader.h"
-#import "XmlElement.h"
 #import "MHVResponse.h"
 
 @interface HealthVaultResponse (Private)
 
 -(BOOL) deserializeXml:(NSString *) xml;
 
-/// Initializes the fields using xml string provided.
-/// @param xml - response xml representation.
-- (BOOL)parseFromXml: (NSString *)xml __deprecated; // Use deserializeXml instead
-
 /// Retrieves info section from xml.
 /// Info section is represented by <wc:info> xml element.
 /// @param xml - xml from which to retrieve info section.
```

Comment:
There is some code that is on line 42 that is commented out that can now be deleted.